src/migrations/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = [
   require('./nightwatch'),
   require('./protractor'),
   require('./webdriverio'),
+  require('./ember'),
```

Comment:
ಠ_ಠ

Reply:
Only reasoning is in the stdout list it has aliases listed next to it and so does the migration beneath it. So my logic was grouping migrations that have aliases and this one is at the top of that group.